implementation/src/main/java/io/smallrye/openapi/api/OpenApiConfigImpl.java

```diff
@@ -228,6 +226,14 @@ public boolean applicationPathDisable() {
         return applicationPathDisable;
     }
 
+    /**
+     * getConfig().getOptionalValue(key) can return "" if optional {@link Converter}s are used. Enforce a null value if
+     * we get an empty string back.
+     */
+    private String getStringConfigValue(String key) {
+        return getConfig().getOptionalValue(key, String.class).map(v -> "".equals(v.trim()) ? null : v).orElse(null);
```

Comment:
This is my own ignorance here - but I assume that `map()` is only called if the value is non-null.  Otherwise the call to `trim()` will result in an NPE.